name.abuchen.portfolio/src/name/abuchen/portfolio/model/InvestmentPlan.java

```diff
@@ -19,6 +19,11 @@
 
 public class InvestmentPlan implements Named, Adaptable, Attributable
 {
+    public enum Type
+    {
+        BUY_OR_DELIVERY, DEPOSIT, REMOVAL
```

Comment:
🤔 ...
könnten wir hier den Standard der Typen vielleicht beibehalten statt `BUY_OR_DELIVERY` ?
https://github.com/buchen/portfolio/blob/86e947d596dcc5d719973f1015725e4daf7fad00/name.abuchen.portfolio/src/name/abuchen/portfolio/model/PortfolioTransaction.java#L17

Was meinst du?!

Reply:
Ich wollte nicht den ganzen vorhandenen Code umbauen. Aktuell (und auch schon vor meiner Erweiterung) werden Buy und Delivery in einem Dialog bei den Investment-Plänen abgehandelt.